README.md

```diff
@@ -0,0 +1,16 @@
+# Menge
+
+
+## Mac OSX
+
+Steps to follow in order to get `Menge` compiled:
```

Comment:
You might want to mention dependencies here? I used `homebrew` to get it working on mine.